lib/metadata_json_lint.rb

```diff
@@ -5,11 +5,7 @@
 
 module MetadataJsonLint
   def options
-    @options ||= {
-      :fail_on_warnings    => true,
-      :strict_license      => true,
-      :strict_dependencies => false
-    }
+    @options ||= Struct.new(:fail_on_warnings, :strict_license, :strict_dependencies).new(true, true, false)
```

Comment:
I guess I'm curious what the benefit of Struct.new() is over a hash, since it's not parsing user input here that could be invalid.